core-web/libs/portlets/edit-ema/portlet/src/lib/shared/enums.ts

```diff
@@ -54,3 +54,8 @@ export enum FormStatus {
     SAVED = 'SAVED',
     PRISTINE = 'PRISTINE'
 }
+
+export enum EDITOR_MODE {
+    PREVIEW = 'preview',
+    EDIT = 'edit'
+}
```

Comment:
I think `UVE_MODE` is more accurate, the editor is the edit mode

Reply:
agree, changed